src/iguana/algorithms/meson.build

```
@@ -52,6 +52,11 @@ algo_dict += [
     'has_validator': false,
     'test_args': { 'banks': ['REC::Particle'] },
   },
+  {
+    'name': 'clas12::FiducialFilter',
+    'has_validator': true,
+    'test_args': { 'banks': ['REC::Particle', 'REC::Traj', 'RUN::config'] },
+  },
```

Don't worry about Fortran bindings for now, I need to re-think how they are done anyway.
```suggestion
  {
    'name': 'clas12::FiducialFilter',
    'has_validator': true,
    'has_c_bindings': false,
    'test_args': { 'banks': ['REC::Particle', 'REC::Traj', 'RUN::config'] },
  },
```

---

(This should fix the failing CI tests)